### PR TITLE
Fixes #6880

### LINKED ISSUE #6880: Mulebot running over Vox creates two (2) colors of blood
image

### PR DESCRIPTION
**Pull request by ghost**
Fixes #6880

### CODE REVIEW

File: code/game/machinery/bots/mulebot.dm [hunk 1]
```diff
@@ -757,7 +757,11 @@ var/global/mulebot_count = 0
 	H.apply_damage(0.5*damage, BRUTE, "l_arm")
 	H.apply_damage(0.5*damage, BRUTE, "r_arm")
 	bloodiness += 4
-	currentBloodColor=bloodcolor // For if species get different blood colors.
```
Comment (clusterfack) [inline code/game/machinery/bots/mulebot.dm]: Why is bloodcolor not sufficient

Comment (ghost) [inline code/game/machinery/bots/mulebot.dm]: I don't know what you mean

Comment (Konater) [inline code/game/machinery/bots/mulebot.dm]: shouldn't it be bloodcolour?

Comment (clusterfack) [inline code/game/machinery/bots/mulebot.dm]: Why is bloodcolor not being passed the correct argument

Comment (Intigracy) [inline code/game/machinery/bots/mulebot.dm]: He's saying

 /obj/machinery/bot/mulebot/proc/RunOverCreature(var/mob/living/H,var/bloodcolor)

Why is the mob's species color not being passed as bloodcolor? You shouldn't need currentBloodColor at all.

**Pull request merged**